src/utility/arrayUtils.js

```diff
@@ -18,13 +18,13 @@ export function findPos(arr, obj) {
 	return false;
 }
 
-/* removePos(arr, obj)
+/** removePos
  * Remove an object in arr based on its pos attribute.
  *
- * arr: Array: The array to look for obj in.
- * obj: Object: Anything with pos attribute. Could be Hex of Creature.
+ * @param {array} arr: The array to look for obj in.
```

Comment:
Does eslint not expect Array, similar to Object here?

Reply:
eslint doesn't complain